lib/ts/user.ts

```diff
@@ -74,6 +76,7 @@ export class LoginMethod implements RecipeLevelUser {
             phoneNumber: this.phoneNumber,
             thirdParty: this.thirdParty,
             timeJoined: this.timeJoined,
+            timeBanned: this.timeBanned,
```

Comment:
dont think we need to return this boolean to the frontend (this function is used to serialise the user object to return to the frontend).